_specifications/lsp/3.18/workspace/symbol.md

```diff
@@ -104,6 +104,12 @@ interface WorkspaceSymbolParams extends WorkDoneProgressParams,
 	/**
 	 * A query string to filter symbols by. Clients may send an empty
 	 * string here to request all symbols.
+	 *
+	 * The `query`-parameter should be interpreted in a *relaxed way* as editors
+	 * will apply their own highlighting and scoring on the results. A good rule
```

Comment:
Could this be changed to `as editors can apply...` ? Not all editors do this and I don't see why they'd need to?